iset.mm

```diff
@@ -72370,6 +72372,21 @@ Similar to the Axiom of Choice (first form) of [Enderton] p. 49.
 =-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=
 $)
 
+  $c Ap $.
+
+  $( Apartness predicate symbol. $)
+  wap $a wff R Ap A $.
+
+  ${
+    $d x y z A $.  $d x y z R $.
+    $( Apartness predicate.  A relation ` R ` is an apartness if it is
+       irreflexive, symmetric, and cotransitive.  (Contributed by Jim Kingdon,
+       14-Feb-2025.) $)
+    df-pap $a |- ( R Ap A <-> ( ( R C_ ( A X. A ) /\ A. x e. A -. x R x ) /\
+      ( A. x e. A A. y e. A ( x R y -> y R x ) /\
+      A. x e. A A. y e. A A. z e. A ( x R y -> ( x R z \/ y R z ) ) ) ) ) $.
```

Comment:
That property would rather be called "co-left Euclidean".

Co-transitive is: `( x R y -> ( x R z \/ z R y ) )` 

(Of course it's equivalent for symmtric relations.)

Reply:
Given that we also have symmetry in the same definition, :shrug: , but I think in one of the proofs I had to reverse `y R z` using symmetry so I suppose that could be telling us something in case we do ever want to do something with similar relations which don't assume symmetry.

Reply:
If you prefer to use "co-left Euclidean", then use that term in the comment, and more importantly, remove the symmetry assumption since it is redundant.